### PR TITLE
test ids for browser images

### PR DESCRIPTION
**Pull request by Khvoya**

### CODE REVIEW

File: packages/browser/src/MediaOutput.tsx [hunk 1]
```diff
@@ -47,7 +47,7 @@ export const MediaOutput = ({
         }
     }, [onOutput]);
     return (
-        <View onLayout={onLayout}>
+        <View testID={`media_${prompt || 'default'}`} onLayout={onLayout}>
```
Comment (savelichalex) [inline packages/browser/src/MediaOutput.tsx]: Do we really need prompt here? It's optional, and you still might be in a situation, when there will be few elements with the same `testID`. Besides, it would be hard to follow whether someone changed it or not

Comment (Khvoya) [reply packages/browser/src/MediaOutput.tsx]: Suggest something else as a unique identifier

Comment (savelichalex) [reply packages/browser/src/MediaOutput.tsx]: Maybe ask debot to provide it? Only debot can provide unique.
Or we can use image url.

**Review by savelichalex — APPROVED** [review]

File: packages/chats/src/Lightbox/DuplicateContent.tsx [hunk 2]
```diff
@@ -169,7 +169,10 @@ export const DuplicateContent = ({
                             <View onLayout={onLayout}>{previewImage}</View>
                         </Animated.View>
                         {isFullSizeDisplayed ? (
-                            <Animated.View style={styles.fullSizeImage}>
+                            <Animated.View
+                                testID={`image_expanded`}
```
Comment (sertony) [inline packages/chats/src/Lightbox/DuplicateContent.tsx]: You ESLint seem not to work properly in `surf@6.4.4` ;)
```suggestion
                                testID="image_expanded"
```

**Review by sertony — APPROVED** [review]
